DisguiseUnityRenderStream/Runtime/PipelineAbstraction/DepthCopy.cs

```diff
@@ -0,0 +1,193 @@
+using System;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.Rendering;
+
+namespace Disguise.RenderStream
+{
+    /// <summary>
+    /// Copies a camera's depth into a texture.
+    /// </summary>
+    class DepthCopy
+    {
+#if HDRP_13_1_8_OR_NEWER
+        const string k_ShaderName = "Hidden/Disguise/RenderStream/DepthCopyHDRP";
+#elif URP_13_1_8_OR_NEWER
+        const string k_ShaderName = "Hidden/Disguise/RenderStream/DepthCopyURP";
+#endif
+        
+        /// <summary>
+        /// Defines how to transform the depth values into the 0-1 range.
+        /// The modes are equivalent to the depth sampling modes of the Shader Graph's Scene Depth Node:
+        /// https://docs.unity3d.com/Packages/com.unity.shadergraph@15.0/manual/Scene-Depth-Node.html.
+        /// </summary>
+        public enum Mode
+        {
+            Raw,
+            Eye,
+            Linear01
+        }
+        
+        /// <summary>
+        /// Parameters for the depth copy.
+        /// </summary>
+        public struct FrameData
```

Comment:
Do we need this `struct` type? It seems to have only 1 real member and a trivial property.